dev/sg/install.sh

```diff
@@ -10,23 +10,21 @@ echo "Compiling..."
 #           it rw we prevent failing go install.
 go install -mod=mod .
 
-# Let's figure out where this got installed. First, we need to calculate the
-# effective $GOBIN; this logic is documented at
-# https://golang.org/cmd/go/#hdr-Compile_and_install_packages_and_dependencies
-orig_gobin="${GOBIN-}"
-if [ -z "$orig_gobin" ]; then
-  if [ -z "${GOPATH:-}" ]; then
-    GOBIN="$HOME/go/bin"
-  else
-    GOBIN="$GOPATH/bin"
-  fi
-fi
+# Let's find the install target. The documentation at
+#   https://golang.org/cmd/go/#hdr-Compile_and_install_packages_and_dependencies
+# that describes the effective $GOBIN path that's used doesn't seem correct
+# when a tool like `asdf` is used to manage Go installations.
+
+# So we use what the Go documentation recommends here
+#   https://golang.org/doc/tutorial/compile-install
+# and find the install target
+target=$(go list -f '{{.Target}}')
```

Comment:
Spaces would be a problem here:

```suggestion
target="$(go list -f '{{.Target}}')"
```